src/scss/_variables.scss

```diff
@@ -199,13 +199,13 @@ $brand-metallic-lightest-color: $black;
 // Custom brand color alternates that meet white-background contrast requirements.
```

Comment:
Yeah, it was meant to stand for "Accessible over White" or "Alternate over White"; I'll add a note about that in the comment.